libkbfs/keybase_daemon_rpc.go

```diff
@@ -308,9 +315,35 @@ func (k *KeybaseDaemonRPC) ShouldRetryOnConnect(err error) bool {
 	return !inputCanceled
 }
 
+func (k *KeybaseDaemonRPC) keepAliveLoop(ctx context.Context) {
+	// We need to send another HelloIAm message as soon as the
+	// connection is re-established.  Unfortunately OnConnect() isn't
+	// called until KBFS makes another outgoing RPC, which might not
+	// happen for a while.  So continuously send a cheap RPC in the
+	// background, so that OnConnect will get called as soon as the
+	// connection comes back.
+	ticker := time.NewTicker(1 * time.Second)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-ticker.C:
+			const sessionID = 0
+			_, err := k.sessionClient.CurrentSession(ctx, sessionID)
```

Comment:
@maxtaco @patrickxb: Is `CurrentSession` cheap enough to use as a keep-alive ping?  If not, is there a better choice?